index.html

```diff
@@ -80,6 +80,10 @@
         <button data-action="log-data">Log segments/points</button>
       </div>
 
+      <ul id="segments">
```

Comment:
@chrisn What info should be displayed for the segments and points when outputted to the page?